Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM alpine:edge
+FROM codeclimate/alpine-ruby:b36
```

Comment:
I believe we only have `b36` publicly atm https://hub.docker.com/r/codeclimate/alpine-ruby/tags/

```
$ docker pull codeclimate/alpine-ruby:b38
Pulling repository codeclimate/alpine-ruby
FATA[0001] Tag b38 not found in repository codeclimate/alpine-ruby 
```